addon/components/offering-manager.js

```diff
@@ -12,11 +12,19 @@ export default class OfferingManagerComponent extends Component {
   @tracked showRemoveConfirmation = false;
   @tracked hoveredGroups = [];
 
+  foo = undefined;
+
   @use learnerGroups = new ResolveAsyncValue(() => [this.args.offering.learnerGroups]);
   @use session = new ResolveAsyncValue(() => [this.args.offering?.session]);
   @use course = new ResolveAsyncValue(() => [this.session?.course]);
+  @use courseStartDate = new ResolveAsyncValue(() => [this.course.startDate]);
+  @use courseEndDate = new ResolveAsyncValue(() => [this.course.endDate]);
```

Comment:
`start/endDate` on course aren't async. These can be getters
`get courseStartDate(){ return this.course?.startDate; }`
or probably even better just be `{{this.course.startDate}}` in the template.

Reply:
:heavy_check_mark: